styles/modules/_form.scss

```diff
@@ -0,0 +1,21 @@
+.form {
+    &--compact {
+        @include mq($screen-s) {
+            display: flex;
+            justify-content: center;
+        }
+        .form__heading {
+            line-height: $button-height;
+            margin-right: 20px;
+        }
+        .form__content {
+            display: flex;
+        }
+    }
+
+    div.mage-error[generated] {
```

Comment:
Apart of this selector, move it to components, if you really thinks it's necessary.